### DISCUSSION TITLE
새 UI 적용 어떻게 하는 걸까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#1223] [5. Q&A]
![image](https://user-images.githubusercontent.com/5895396/186361391-cc70f823-429b-4440-b0ff-98ca1c4bfe72.png)
이쪽이 아마 새로 변경된 UI 같은데, 적용 방법을 모르겠습니다.
아시는 분 있으실까요?

![image](https://user-images.githubusercontent.com/5895396/186361587-8b36705c-57c0-458d-a6b9-a7f24d5416f5.png)
지금은 계속 이 UI 사용중입니다.

**Comment by zlmrzlo** [comment]
혹시 유니티 패키지에서 제페토 패키지들 업데이트 하셨는지 확인 가능할까요?
웬만하면 패키지 업데이트하면 될 것 같아요.

**Reply by ryusiin** [reply]
![image](https://user-images.githubusercontent.com/5895396/186552481-deb8dc74-0283-4da4-bc0c-19a03a8b5612.png)
일단 모든 업데이트를 했는데 안되네요...

**Comment by koldshine** [comment]
저 UI적용이 이제 가능해진건가요? 예전에 질문드렸을때엔 공식월드만 가능하다고 답변 받아서요...

**Reply by hwisulee** [reply]
저도 마찬가지로 지원 안되는 걸로 알고 있습니다.

**Reply by ryusiin** [reply]
공식 월드만 되는 건가요...

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 유니티로 제작하는 월드는 크리에이터분들이 월드 UI 등으로 다양하게 활용하실 수 있도록 홈, 채팅 UI만 기본적으로 생성됩니다.
현재 API로 제공하는 UI는 다음과 같습니다.

- 카메라 https://studio.zepeto.me/guides/screenshot
- 동영상 촬영 https://studio.zepeto.me/guides/video-save-share-post
- 퀵채팅 https://studio.zepeto.me/guides/quick-message

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
공식 UI 관련 질문이 자주 올라와서 FAQ에 정리해서 올려두었습니다 💪
https://studio.zepeto.me/guides/world-faqs
